outputs/atlas_account/output.facets.yaml

```
@@ -0,0 +1,22 @@
+        project_id:
+          required: true
+          type: string
+          description: The ID of the Atlas project
+        public_key:
+          required: true
+          type: string
+          description: The public key for the Atlas account
+        private_key:
+          required: true
+          type: string
+          description: The private key for the Atlas account
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)